# Movable Ingredient
bacon: # scale 1
  thickness: [0.0005, 0.005]
  scale: [1, 1, 1]
  file_name: "bacon.stl"

turkey: # scale 1
  # thickness: [0.006, 0.009]
  thickness: [0.007, 0.002]
  scale: [1, 1, 1]
  file_name: "turkey.stl"

bread: # scale 1
  thickness: [0.0085, 0.0085]
  scale: [0.85, 0.85, 0.85]
  file_name: "bread.stl"

garlic_bread: # scale 1
  thickness: [0.0085, 0.0085]
  scale: [0.85, 0.85, 0.85]
  file_name: "garlic_bread.stl"

cheese: # scale 1
  thickness: [0.0055, 0.0055]
  scale: [1, 1, 1]
  file_name: "cheese.stl"

feta_cheese: # scale 1
  thickness: [0.0055, 0.0055]
  scale: [1, 1, 1]
  file_name: "feta_cheese.stl"

ham: # scale 0.5
  thickness: [0.005, 0.005]
  scale: [0.5, 0.5, 0.5]
  file_name: "ham.stl"

tomato: #scale 0.2
  # thickness: [0.024, 0.0315]
  thickness: [0.0315, 0.024]
  scale: [0.3, 0.3, 0.3]
  file_name: "tomato.stl"

butter:
  thickness: [0, 0.02]
  scale: [1, 1, 1]
  file_name: "butter.stl"

lettuce:
  thickness: [0, 0.01]
  scale: [0.8, 0.8, 0.8]
  file_name: "lettuce.stl"

escarole:
  thickness: [0, 0.01]
  scale: [0.8, 0.8, 0.8]
  file_name: "escarole.stl"

dill:
  thickness: [0, 0.01]
  scale: [0.8, 0.8, 0.8]
  file_name: "dill.stl"

mango:
  # thickness: [0.024, 0.0315]
  thickness: [0.0315, 0.024]
  scale: [0.3, 0.3, 0.3]
  file_name: "mango.stl"

avocado:
  # thickness: [0.024, 0.0315]
  thickness: [0.0315, 0.024]
  scale: [0.3, 0.3, 0.3]
  file_name: "avocado.stl"

cucumber: # scale 1
  # thickness: [0.01, 0.01]
  thickness: [0.013, 0.01]
  scale: [1, 1, 1]
  file_name: "cucumber.stl"

sweet_pepper: # scale 1
  thickness: [0.019, 0.028]
  scale: [1, 1, 1]
  file_name: "sweet_pepper.stl"

green_pepper: # scale 1
  thickness: [0.019, 0.028]
  scale: [1, 1, 1]
  file_name: "green_pepper.stl"

onion: # scale 3
  # thickness: [0.036, 0.016]
  thickness: [0.025, 0.037]
  scale: [3, 3, 3]
  file_name: "onion.stl"

potato:
  # thickness: [0.0134, 0.0134]
  thickness: [0.014, 0.014]
  scale: [1.3, 1.3, 1.3]
  file_name: "potato.stl"

banana: # scale 1
  thickness: [0.0125, 0.0125]
  scale: [1, 1, 1]
  file_name: "banana.stl"

black_olive: # scale 1
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "black_olive.stl"

walnut: # scale 1
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "walnut.stl"

orange_pepper: # scale 1
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "walnut.stl"

peanut: # scale 1
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "walnut.stl"

shrimp: # scale 1
  thickness: [0.009, 0.009]
  scale: [1, 1, 1]
  file_name: "shrimp.stl"

oregano: # scale 1
  thickness: [0.016, 0.01]
  scale: [1, 1, 1]
  file_name: "oregano.stl"

cilantro: # scale 1
  # thickness: [0.016, 0.02]
  thickness: [0.0, 0.01]
  scale: [1, 1, 1]
  file_name: "cilantro.stl"

cabbage: # scale 1
  # thickness: [0.016, 0.02]
  thickness: [0.0, 0.01]
  scale: [1, 1, 1]
  file_name: "cabbage.stl"

scallion: # scale 1
  thickness: [0.0047, 0.0043]
  scale: [1, 1, 1]
  file_name: "scallion.stl"

spinach: # scale 1
  thickness: [0.0047, 0.0043]
  scale: [1, 1, 1]
  file_name: "spinach.stl"

celery: # scale 1
  thickness: [0.0047, 0.0043]
  scale: [1, 1, 1]
  file_name: "celery.stl"

mayonnaise_bottle: # scale 0.5
  thickness: [0.057, 0.09]
  scale: [0.5, 0.5, 0.5]
  file_name: "mayonnaise_bottle.stl"

lemon_juice_bottle: # scale 0.5
  thickness: [0.057, 0.09]
  scale: [0.5, 0.5, 0.5]
  file_name: "lemon_juice_bottle.stl"

olive_oil_bottle: # scale 1
  thickness: [0.0542, 0.0822]
  scale: [1, 1, 1]
  file_name: "olive_oil_bottle.stl"

sugar_bottle: # scale 0.3
  thickness: [0.042, 0.056]
  scale: [0.3, 0.3, 0.3]
  file_name: "sugar_bottle.stl"

salt_bottle: # scale 0.3
  thickness: [0.042, 0.056]
  scale: [0.3, 0.3, 0.3]
  file_name: "salt_bottle.stl"

pepper_bottle: # scale 0.3
  thickness: [0.042, 0.056]
  scale: [0.3, 0.3, 0.3]
  file_name: "pepper_bottle.stl"

relish_bottle: # scale 0.3
  thickness: [0.042, 0.056]
  scale: [0.3, 0.3, 0.3]
  file_name: "relish_bottle.stl"

parsley_bottle: # scale 0.3
  thickness: [0.042, 0.056]
  scale: [0.3, 0.3, 0.3]
  file_name: "parsley_bottle.stl"

black_pepper_bottle: # scale 0.3
  thickness: [0.042, 0.056]
  scale: [0.3, 0.3, 0.3]
  file_name: "black_pepper_bottle.stl"

tuna_can:
  thickness: [0, 0.03]
  scale: [1, 1, 1]
  file_name: "tuna_can.stl"

tuna_sandwich:
  thickness: [0.03, 0.027]
  scale: [1, 1, 0.5]
  file_name: "tuna_sandwich.stl"

chopped_tuna_sandwich:
  thickness: [0.03, 0.027]
  scale: [1, 1, 0.5]
  file_name: "chopped_tuna_sandwich.stl"

# Movable Tool
knife: # scale 1
  thickness: [0.0045, 0.0045]
  scale: [1, 1, 1]
  file_name: "knife.stl"

spatula: # scale 1 / blender 1
  thickness: [0.0017, 0.0017]
  scale: [1, 1, 1]
  file_name: "spatula.stl"

scooper: # scale 1 / blender 1
  thickness: [0.004, 0.006]
  scale: [1, 1, 1]
  file_name: "scooper.stl"

spreader: # scale 1 / blender 1
  thickness: [0.0046, 0.0046]
  scale: [1, 1, 1]
  file_name: "spreader.stl"

# Movable Container
bowl: # scale 1
  thickness: [0, 0.08]
  scale: [0.85, 0.85, 0.85]
  file_name: "bowl.stl"

plate: # scale 1
  # thickness: [0, 0.005]
  thickness: [0, 0.024]
  scale: [1, 1, 1]
  file_name: "plate.stl"

pan: # scale 0.8
  # thickness: [0.018, 0.015]
  # thickness: [0, 0.08]
  thickness: [0, 0.1]
  scale: [0.6, 0.6, 0.6]
  file_name: "pan.stl"

pot: # scale 0.7
  # thickness: [0, 0.012]
  thickness: [0, 0.145]
  scale: [0.7, 0.7, 0.7]
  file_name: "pot.stl"

cutting_board: # scale 1
  thickness: [0, 0.015]
  scale: [0.65, 0.65, 0.65]
  file_name: "cuttingboard.stl"
  temp: 0.6

#####################
# Chop/chopped object
chopped_onion:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "chopped_onion.stl"

chopped_cucumber:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_cucumber.stl"

chopped_sweet_pepper:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_sweet_pepper.stl"

chopped_green_pepper:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_green_pepper.stl"

chopped_feta_cheese:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "chopped_feta_cheese.stl"

chopped_cheese:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "chopped_cheese.stl"

chopped_tomato:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_tomato.stl"

chopped_potato:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_potato.stl"

chopped_oregano: # scale 1
  thickness: [0.016, 0.01]
  scale: [1, 1, 1]
  file_name: "chopped_oregano.stl"

chopped_cilantro:
  # thickness: [0.016, 0.02]
  thickness: [0.0, 0.02]
  scale: [1, 1, 1]
  file_name: "chopped_cilantro.stl"

chopped_cabbage:
  # thickness: [0.016, 0.02]
  thickness: [0.0, 0.02]
  scale: [1, 1, 1]
  file_name: "chopped_cabbage.stl"

chopped_shrimp:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "chopped_shrimp.stl"

chopped_mango:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_mango.stl"

chopped_dill:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_dill.stl"

chopped_avocado:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_avocado.stl"

chopped_scallion:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_scallion.stl"

chopped_spinach:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_spinach.stl"

chopped_celery:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_celery.stl"

chopped_banana:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_banana.stl"

chopped_tuna:
  thickness: [0, 0.007]
  scale: [1, 1, 1]
  file_name: "chopped_tuna.stl"

chopped_lettuce:
  thickness: [0, 0.01]
  scale: [0.8, 0.8, 0.8]
  file_name: "chopped_lettuce.stl"

# Spread/spread object
spread_butter:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "spread_butter.stl"

# Pour/bottled object 
mayonnaise:
  thickness: [0, 0.001]
  scale: [1, 1, 1]
  file_name: "mayonnaise.stl"

salt:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "salt.stl"

sugar:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "sugar.stl"

pepper:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "pepper.stl"

parsley:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "parsley.stl"

relish:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "relish.stl"

olive_oil:
  thickness: [0, 0.001]
  scale: [1, 1, 1]
  file_name: "olive_oil.stl"

lemon_juice:
  thickness: [0, 0.001]
  scale: [1, 1, 1]
  file_name: "lemon_juice.stl"

black_pepper:
  thickness: [0, 0.002]
  scale: [1, 1, 1]
  file_name: "black_pepper.stl"

tuna:
  thickness: [0, 0.03]
  scale: [1, 1, 1]
  file_name: "tuna.stl"

# Stir/new object
tuna_spread:
  thickness: [0, 0.005]
  scale: [1, 1, 1]
  file_name: "tuna_spread.stl"